src/js/Alert.js

```diff
@@ -14,6 +68,36 @@ export default class Alert extends React.Component {
                  ? (<StaticIcon class="alert-icon" image={icon.value} />)
                  : (<div className="alert-icon"><Image class="icon" image={icon.value} isTemplate={icon.isTemplate} fillColor={fill} /></div>);
 
+        let speakCheckbox = undefined;
+        // if (this.state.ifSpeak) {
+        //     speakCheckbox = ( 
+        //         <>
+        //             <p>
+        //             <input
+        //                 name="Speak" 
+        //                 type="checkbox"
+        //                 defaultChecked={this.state.speakChecked}
+        //                 onChange={() => this.setState(prevState => ({speakChecked: !prevState.speakChecked}))} />
+        //             <label>Speak</label>
+        //             </p>
+        //         </>
+        //     );
+        // }
+
+        let speakCounter = undefined;
+        // if (this.state.ifSpeak) {
+        //     speakCounter =
+        //     <>
+        //         <p>TTS.Speak: {this.state.speakCounter}</p>
+        //     </>
+        // }
+        let resetTimeoutHTML = store.getState().ui[store.getState().activeApp].alert.softButtons !== undefined ?
+        undefined :
+        <><div className="timeout-box">
```

Comment:
Looks like there is extra <> here

Reply:
Thank you for the comment. Fixed [here](https://github.com/LuxoftSDL/generic_hmi/pull/17/commits/3ae89b18e8553f3df1802cc93eab9b9a5a77af3f).